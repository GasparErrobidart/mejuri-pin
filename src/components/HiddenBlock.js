import styled from 'styled-components'

const HiddenBlock = styled.div`
  height:1px;
  opacity:0.0;
`
export default HiddenBlock
