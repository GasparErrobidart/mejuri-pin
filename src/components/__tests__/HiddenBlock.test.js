import '@testing-library/jest-dom'
import 'jest-styled-components'

import React         from 'react'
import { cleanup }   from '@testing-library/react'
import renderer      from 'react-test-renderer'

import HiddenBlock   from '../HiddenBlock'


afterEach(cleanup)

test('Renders "HiddenBlock" component', () => {
  const testText        = "HiddenBlock Test Text"
  const component       = renderer.create(<HiddenBlock>{testText}</HiddenBlock>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
