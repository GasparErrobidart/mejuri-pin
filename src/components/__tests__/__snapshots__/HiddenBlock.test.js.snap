// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "HiddenBlock" component 1`] = `
.c0 {
  height: 1px;
  opacity: 0.0;
}

<div
  className="c0"
>
  HiddenBlock Test Text
</div>
`;
